core/templates/dev/head/profile/Profile.js

```diff
@@ -19,15 +19,21 @@
  */
 
 oppia.controller('Profile', [
-  '$scope', '$http', '$rootScope',
-  function($scope, $http, $rootScope) {
+  '$scope', '$http', '$rootScope', 'oppiaDatetimeFormatter',
+  function($scope, $http, $rootScope, $oppiaDatetimeFormatter) {
```

Comment:
I'm trying to run the local server to see any console errors, and I'm actually receiving a binding error that results in no connection. Here is the stack traceback:

Starting GAE development server
INFO     2015-11-15 21:28:13,690 sdk_update_checker.py:229] Checking for updates to the SDK.
INFO     2015-11-15 21:28:13,950 api_server.py:190] Starting API server at: http://localhost:59067
INFO     2015-11-15 21:28:13,954 dispatcher.py:192] Starting module "default" running at: http://0.0.0.0:8181
INFO     2015-11-15 21:28:13,955 api_server.py:615] Applying all pending transactions and saving the datastore
INFO     2015-11-15 21:28:13,955 api_server.py:618] Saving search indexes
Traceback (most recent call last):
  File "/Users/kerrywang/oppia_tools/google_appengine_1.9.19/google_appengine/dev_appserver.py", line 83, in <module>
    _run_file(__file__, globals())
  File "/Users/kerrywang/oppia_tools/google_appengine_1.9.19/google_appengine/dev_appserver.py", line 79, in _run_file
    execfile(_PATHS.script_file(script_name), globals_)
  File "/Users/kerrywang/oppia_tools/google_appengine_1.9.19/google_appengine/google/appengine/tools/devappserver2/devappserver2.py", line 1002, in <module>
    main()
  File "/Users/kerrywang/oppia_tools/google_appengine_1.9.19/google_appengine/google/appengine/tools/devappserver2/devappserver2.py", line 995, in main
    dev_server.start(options)
  File "/Users/kerrywang/oppia_tools/google_appengine_1.9.19/google_appengine/google/appengine/tools/devappserver2/devappserver2.py", line 803, in start
    admin.start()
  File "/Users/kerrywang/oppia_tools/google_appengine_1.9.19/google_appengine/google/appengine/tools/devappserver2/admin/admin_server.py", line 116, in start
    super(AdminServer, self).start()
  File "/Users/kerrywang/oppia_tools/google_appengine_1.9.19/google_appengine/google/appengine/tools/devappserver2/wsgi_server.py", line 315, in start
    self._start_all_fixed_port(host_ports)
  File "/Users/kerrywang/oppia_tools/google_appengine_1.9.19/google_appengine/google/appengine/tools/devappserver2/wsgi_server.py", line 352, in _start_all_fixed_port
    raise BindError('Unable to bind %s:%s' % self.bind_addr)
google.appengine.tools.devappserver2.wsgi_server.BindError: Unable to bind localhost:8000